clientSrc/app.less

```diff
@@ -1,3 +1,69 @@
 body {
+    background: linear-gradient(to right, #348f50, #56b4d3);
+    font-family: 'Ubuntu', sans-serif;
+}
+
+/*///////////////////////////////////////////////////////////////////////////////////////////////////////////////*/
+/*NAVBAR STYLING*/
+.navbar {
```

Comment:
Instead of having all the css in one file, you can create individual css/less files for each component.